Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># This is a sample Python script.\r\n\r\n# Press Shift+F10 to execute it or replace it with your code.\r\n# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.\r\n\r\n\r\ndef print_hi(name):\r\n    # Use a breakpoint in the code line below to debug your script.\r\n    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.\r\n\r\n\r\n# Press the green button in the gutter to run the script.\r\nif __name__ == '__main__':\r\n    print_hi('Parm')\r\n\r\n# See PyCharm help at https://www.jetbrains.com/help/pycharm/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 6aeac9b83c4d04681ecdc8161b925549c048bad4)
+++ b/main.py	(date 1647012764982)
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
 # This is a sample Python script.
 
 # Press Shift+F10 to execute it or replace it with your code.
@@ -14,3 +15,9 @@
     print_hi('Parm')
 
 # See PyCharm help at https://www.jetbrains.com/help/pycharm/
+=======
+def hello(name):
+    print(f'Hello, {name}')
+
+hello('Pycharm')
+>>>>>>> 120373b (тест)
